### PR TITLE
Fix some small bugs.

### PR DESCRIPTION
**Pull request by ipcjs**

### CODE REVIEW

**Review by xclud — CHANGES_REQUESTED** [review]

File: example/lib/main.dart [hunk 1]
```diff
@@ -153,7 +153,7 @@ class _MyHomePageState extends State<MyHomePage> {
                     trackShape: CenteredRectangularSliderTrackShape(),
                   ),
                   child: Slider(
-                    divisions: 361,
+                    divisions: 360,
```
[the remark this answers is not in the document]
Comment (ipcjs) [reply example/lib/main.dart:156]: look at [the document](https://api.flutter.dev/flutter/material/Slider-class.html):
```
For example, if min is 0.0 and max is 50.0 and divisions is 5, then the slider can take on the discrete values 0.0, 10.0, 20.0, 30.0, 40.0, and 50.0.
```
so `(max - min) / divisions == gap`.

if `max = -180, min = 180, gap = 1`, then `divisions = 360`

[the remark this answers is not in the document]
Comment (xclud) [reply example/lib/main.dart:156]: This a more a math problem than a Slider problem/design. There are 361 number in this range.

Take this as an example: There are 11 numbers in [-5 5] range. They are [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5].

[the remark this answers is not in the document]
Comment (ipcjs) [reply example/lib/main.dart:156]: see it's [document](https://api.flutter.dev/flutter/material/Slider/divisions.html)
The "divisons" property means "The number of discrete divisions.", not "the number of numbers in the range".

[the remark this answers is not in the document]
Comment (xclud) [reply example/lib/main.dart:156]: You are right. Thank you. Merging.

**Pull request merged**